Sources/SourceControl/RepositoryManager.swift

```diff
@@ -206,10 +206,10 @@ public class RepositoryManager: Cancellable {
                 return handle
             }
             // Update the repository when it is being looked up.
-            let start = DispatchTime.now()
```

Comment:
Moving this down to below the delegate call so that its duration isn't counted as part of it.  This is consistent with the other delegate method duration capturing.